civihr_employee_portal/views/views_export/views_mydetails_emails.inc

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
@powdevel why we are not using feature module for this ?

Reply:
@hitesh-jain Same answer on another comment. I'm following patterns in `civihr_employee_portal`.